src/source.cc

```diff
@@ -103,7 +103,10 @@ std::set<cyclus::BidPortfolio<cyclus::Material>::Ptr> Source::GetMatlBids(
 
     // calculate packaging
     double bid_qty = context()->GetPackage(package)->GetFillMass(qty);
-    int n_full_bids = static_cast<int>(std::floor(qty / bid_qty));
+    int n_full_bids = 0;
+    if (bid_qty >= context()->GetPackage(package)->fill_min()) {
```

Comment:
In `mat_sell_policy` we only make sure that it's not 0 rather than compare to `fill_min`.  I wonder which one is more robust and more useful?

Reply:
Perhaps we should also somehow return `n_full_bids` from `GetFillMass()` and handle this in a single robust way?

Reply:
Fixing in [cyclus#1790](https://github.com/cyclus/cyclus/pull/1790)